rest-api-spec/src/main/resources/rest-api-spec/test/search.aggregation/140_value_count_metric.yml

```diff
@@ -0,0 +1,173 @@
+setup:
+  - do:
+      indices.create:
+          index: test_1
+          body:
+            settings:
+              number_of_replicas: 0
+            mappings:
+              test:
+                properties:
+                  int_field:
+                     type : integer
+                  double_field:
+                     type : double
+                  string_field:
+                     type: keyword
+  - do:
+       bulk:
+         refresh: true
+         body:
+           - index:
+               _index: test_1
+               _type:  test
```

Comment:
Oh good call, completely forgot about that.  I'll edit the prior tests that I merged too, since they use the same setup.